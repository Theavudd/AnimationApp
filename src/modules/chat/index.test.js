// import Counter from '.';
// import React from 'react';
// import {render} from '@testing-library/react';

// describe('Counter', () => {
//   it('Render Correct initial count', () => {
//     const {getByTestId} = render(<Counter />);
//     const countValue = getByTestId('Count').testContent;
//     console.log('countValue', countValue);
//     expect(countValue).toBe('0');
//   });
// });

import React from 'react';
import renderer from 'react-test-renderer';
import Counter from '.';

test('renders correctly', () => {
  const {} = renderer.create(<Counter />);
  console.log('getIn', getInstance().children());
});
