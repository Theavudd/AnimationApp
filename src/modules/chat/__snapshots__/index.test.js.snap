// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView>
  <Text
    data-testID="Count"
  >
    0
  </Text>
  <View
    accessibilityRole="button"
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        [
          {},
        ]
      }
    >
      <Text
        style={
          [
            {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        increment
      </Text>
    </View>
  </View>
  <View
    accessibilityRole="button"
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        [
          {},
        ]
      }
    >
      <Text
        style={
          [
            {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Decrement
      </Text>
    </View>
  </View>
</RCTSafeAreaView>
`;
